raiden/waiting.py

```diff
@@ -153,6 +155,60 @@ def wait_for_participant_deposit(
         current_balance = balance(channel_state)
 
 
+def wait_single_channel_deposit(
+    app_deposit: "App",
+    app_partner: "App",
+    registry_address: TokenNetworkRegistryAddress,
+    token_address: TokenAddress,
+    total_deposit: TokenAmount,
+    retry_timeout: float,
+) -> None:
+    wait_for_participant_deposit(
+        raiden=app_deposit.raiden,
+        token_network_registry_address=registry_address,
+        token_address=token_address,
+        partner_address=app_partner.raiden.address,
+        target_address=app_deposit.raiden.address,
+        target_balance=total_deposit,
+        retry_timeout=retry_timeout,
+    )
+    wait_for_participant_deposit(
```

Comment:
Why does it need to wait for two deposits here? And please add docstrings to the functions.

Reply:
Each wait is for the same deposit but makes sure it's seen by each app individually.